tsk/fs/btrfs_csum.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+** The Sleuth Kit
+**
+** Brian Carrier [carrier <at> sleuthkit [dot] org]
+** Copyright (c) 2003-2011 Brian Carrier.  All rights reserved
+**
+** TASK
+** Copyright (c) 2015 Stefan Pöschel.  All rights reserved
+**
+** This software is distributed under the Common Public License 1.0
+*/
+
+/*
+ * Contains the checksum part for Btrfs file system support.
+ */
+
+#include "tsk/base/crc.h"
+#include <stdio.h>
```

Comment:
What here needs `<stdio.h>`?

Reply:
I may have added it for debugging. I'll remove it.

Reply:
fixed in 29968391c